tests/test_check_language.py

```diff
@@ -0,0 +1,15 @@
+from gtrending import check_language
+
+
+def assertions(response, expected):
+    assert isinstance(response, bool)
+    assert response == expected
+
+
+def test_check_language():
+    response = check_language("python")
+    assertions(response, True)
+    response = check_language("javascript")
+    assertions(response, True)
+    response = check_language("false_language")
+    assertions(response, False)
```

Comment:
No need to put assertions logic into separate function simply:
```suggestion
def test_check_language():
    assert check_language("python")
    assert check_language("javascript")
    assert not check_language("false_language")
```

Same thing with `test_check_since.py` and `test_check_spoken_language.py`